support-workers/src/main/scala/com/gu/support/workers/lambdas/Handler.scala

```diff
@@ -36,12 +59,17 @@ abstract class Handler[IN, OUT](implicit
     }
 
   def handleRequestFuture(is: InputStream, os: OutputStream, context: Context): Future[Unit] = {
+    def log[T](message: String, a: T): Unit = {
+      val prettyData = pprint.apply(a).plainText.replaceAll("\\n", "\n\t")
```

Comment:
Interesting that we don't have to `import` `pprint`?

Reply:
`pprint` is a package with an apply method in the package object!  I guess the idea was to make it available everywhere without imports?  You can also log directly to the console with pprint.log, (however in our case we want to send it to our logger)